src/VisualStudio/Core/Def/ServicesVisualStudio_xuzbge3c_wpftmp.csproj

```diff
@@ -0,0 +1,449 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
>Project [](start = 1, length = 7)

This file was probably unintentionally added.

Reply:
Removed